chasm/lib/callback/component.go

```diff
@@ -39,10 +39,15 @@ func (c *Callback) LifecycleState(_ chasm.Context) chasm.LifecycleState {
 	return chasm.LifecycleStateRunning
 }
 
-func (c *Callback) State() *callbackspb.CallbackStatus {
-	return c.status
+func (c *Callback) Status() callbackspb.CallbackStatus {
```

Comment:
Let's wait and see where this rename lands, thanks for incorporating it already though.

Reply:
Got it, I'll remove from this PR to not block this work.